### DISCUSSION TITLE
Show: Custom command to get random daily note

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#429] [Random]
A [gist](https://gist.github.com/minthemiddle/cb59506c2366acc8f168b5bb485293cf) for a [custom command](https://docs.nota.md/custom-commands) to show a random daily note.
The folder for daily notes is hard-coded, I did not find a way to reference it dynamically.

**Comment by astoilkov** [comment]
Nice. Great custom command I would personally use. Thanks!
